apps/web-giddh/src/app/settings/permissions/form/form.component.html

```diff
@@ -1,143 +1,219 @@
-
-                <section class="periods container-fluid mrB1"
-                    *ngIf="permissionForm.get('periodOptions').value === 'daterange'">
-                    <div class="row">
-                        <div class="col-xs-12">
-                            <label>From - To</label>
-                            <input name="DateRangePicker" [bsValue]="dateRangePickerValue"
-                                (bsValueChange)="onSelectDateRange($event)" placeholder="Select Date Range" type="text"
-                                class="form-control" bsDaterangepicker>
-
-                            <!-- <input class="form-control " name="from" formControlName="from" type="text" placeholder="DD-MM-YYYY" /> -->
-                        </div>
-                        <!-- <div class="col-xs-6">
-                <label>To</label>
-                <input class="form-control " name="to" formControlName="to" type="text" placeholder="DD-MM-YYYY" />
-            </div> -->
-                    </div>
-                </section>
-
-                <section class="periods container-fluid mrB1"
-                    *ngIf="permissionForm.get('periodOptions').value === 'pastperiod'">
-                    <label>Select Past Period</label>
-                    <div class="row">
-                        <div class="col-xs-6">
-                            <input class="form-control " name="duration" digitsOnlyDirective formControlName="duration"
-                                type="number" placeholder="Number" />
-                        </div>
-                        <div class="col-xs-6 pdL0 lh34">
-                            Days
-                        </div>
-                    </div>
-                </section>
-
-            </section>
+                    </span>-->
+                </ul>
+            </div>
         </div>
-    </div>
-    <!-- time span end -->
-
-    <div class="form-group" [exclude]="'.ip-addresses-dropdown'" [attachOutsideOnClick]="true" (clickOutside)="methodForToggleSection('rangeSpanSection')">
-        <label *ngIf="isLblShow">IP Addresses</label>
-        <div class="btn-group " [ngClass]="{'open': showIPWrap, 'btn-group-liq': !isHorizntl}">
-            <button (click)="showIPWrap = !showIPWrap" type="button" class="form-control text-left ">
-                {{selectedIPRange}}
-                <span class="pull-right"><span class="fa fa-caret-down"></span></span>
-            </button>
-            <section class="dropdown-menu custom_dropdown ip-addresses-dropdown" role="menu" [ngClass]="{'open': showIPWrap}">
-                <div class="clearfix option_box pd1">
-
-                    <span (click)="isOpenedInModal ? handleIpAddressChange('cidr_range'): ''" class="pd0 mrR3">
+        <!-- time span end -->
+
+        <!--  -->
+        <section class="form-group col-xs-12"
+            *ngIf="permissionForm.get('periodOptions').value === 'daterange'">
+                <label>From - To</label>
```

Comment:
Add to JSON

Reply:
Already add JSON for **From - To** content